ckanext/switzerland/plugin.py

```diff
@@ -39,15 +39,13 @@
 )
 
 
-class OgdchPlugin(plugins.SingletonPlugin):
+class OgdchPlugin(plugins.SingletonPlugin, DefaultTranslation):
     plugins.implements(plugins.IConfigurer)
     plugins.implements(plugins.IValidators)
     plugins.implements(plugins.IFacets)
     plugins.implements(plugins.IActions)
     plugins.implements(plugins.ITemplateHelpers)
 
-    # IConfigurer
-
```

Comment:
Did you remove that comment on purpose?

Reply:
oh noez! Coming back in 3 .. 2 .. 1 .. commit.